detox/android/detox/src/test/java/com/wix/detox/DetoxActionHandlersTest.kt

```diff
@@ -112,8 +114,7 @@ class InvokeActionHandlerTest : DetoxActionHandlerTestBase() {
     // TODO reply with an actual result
     @Test fun `should reply with dummy result data`() {
         uut().handle(params, messageId)
-
-        verify(wsClient).sendAction(any(), argThat { size == 1 && this["result"] == "(null)" }, any())
+        verify(wsClient).sendAction(any(), argThat { size == 1 && this["result"] == null }, any())
     }
```

Comment:
plz also add a concrete test where the invoke action returns an actual result... in tdd we trust